flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -43,18 +44,75 @@ class FlxGamepad implements IFlxDestroyable
 	 */
 	private var axis:Array<Float> = [for (i in 0...6) 0];
 	
-	#if flash
+	#if (flash || next)
 	private var _device:GameInputDevice; 
 	#end
 	
-	public function new(ID:Int, GlobalDeadZone:Float = 0) 
+	#if (flash)
+	private var _isChrome:Bool = false;
+	#end
+	
+	/**
+	 * Helper class to check if a button is pressed.
+	 */
+	public var pressed:FlxGamepadButtonList;
```

Comment:
These should all be `(default, null)` - just realized that isn't even the case in `FlxKeyManager` where you copy-pasted them from. :o